react-app/src/app/screens/Game/components/Square/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import style from './styles.scss';
+
+function Square(props) {
```

Comment:
Deconstruct the props
```suggestion
function Square({value, onClick}) {
```